### DISCUSSION TITLE
Why are there peaks on the photon spectrum, shouldn't they be smooth curves？

### DISCUSSION THREAD

**Discussion by phantasuson** [nrc-cnrc/EGSnrc#769] [Physics] [answered]
6MV electrons beam, source NO.0 with a W700ICRU target. They occur no matter how I change the target size.

![6369f1b98db9dad9115d07cd5ac046f](https://user-images.githubusercontent.com/80371548/133580273-ef05b87a-02b0-4206-8d99-7d5a8724edca.png)

![cf295d52ef4da337dd945eb3db9560f](https://user-images.githubusercontent.com/80371548/133580288-846a3d58-7f5d-4d6e-a3f8-22b232ba947a.png)

**Comment by ftessier** [comment, marked as answer]
These peaks are x-rays emitted when electrons relax between inner shells. Their positions depend on the element, hence they are called "characteristic" x-rays. The smooth background curve is from bremsstrahlung emission. See any textbook on x-rays, or for a brief summary: http://hyperphysics.phy-astr.gsu.edu/hbase/quantum/xrayc.html.

**Reply by phantasuson** [reply]
Thanks, I see. But I didn't expect it to be such a big part compared with other papers. What made the difference?
![1024b6fb5fa3adb9c7eeb4d0235c64b](https://user-images.githubusercontent.com/80371548/133713006-a3044f39-b3c4-4dc3-9f9e-f5be2f4cb35b.png)
![image](https://user-images.githubusercontent.com/80371548/133713096-f8dbd647-dea7-4b21-9eac-4dba758ca112.png)

**Reply by ftessier** [reply]
I don't know; what _IS_ the difference in the models? Difficult to comment without more information; for example, the W target is supported by some other material, there may be filtration, etc. But the spectra for pure W unequivocally has a significant K peak around 58 keV (see https://xdb.lbl.gov/Section1/Periodic_Table/W_Web_data.htm). You can find many references for this, e.g., 

![image](https://user-images.githubusercontent.com/1444241/133780910-8c050163-a0ea-4e69-b247-be3c66e25c2c.png)

from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4958109/
